src/azeventgrid.ngrok.webapi/Controllers/EventGridController.cs

```diff
@@ -29,19 +28,16 @@ public async Task<IActionResult> Post()
                 using (var reader = new StreamReader(Request.Body, Encoding.UTF8))
                 {
                     var jsonContent = await reader.ReadToEndAsync();
-                    var eventGridEvents = JsonConvert.DeserializeObject<List<EventGridEvent>>(jsonContent);
+                    var eventGridEvents = JsonSerializer.Deserialize<List<EventGridEvent>>(jsonContent);
```

Comment:
I believe that we want to prefer using `EventGridEvent.Parse` here.  _([ref](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventgrid/Azure.Messaging.EventGrid/samples/Sample3_ParseAndDeserializeEvents.md#parse-events-from-json-payload))_